central/processindicator/store/postgres/store.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by pg-bindings generator. DO NOT EDIT.
+// Code originally generated by pg-bindings generator.
```

Comment:
nit:  Can we add a TODO with the reference to the ticket(s) we plan to use to get this back to being generated?